tests/manage/pv_services/pv_encryption/test_encrypted_rbd_pvc_snapshot.py

```diff
@@ -56,6 +56,15 @@
                 "v2", kmsprovider, False, marks=pytest.mark.polarion_id("OCS-2613")
             ),
         ]
+    if Version.coerce(config.ENV_DATA["ocs_version"]) >= Version.coerce("4.12"):
```

Comment:
Please use this approach of comparing versions:
https://github.com/red-hat-storage/ocs-ci/blob/de5f02841523d4defa950e7bc09735333e6a09b4/ocs_ci/deployment/baremetal.py#L203

Reply:
Done